docs/_posts/2018-09-19-security-fixes-for-3-6-3-7-3-8.markdown

```diff
@@ -18,6 +18,7 @@ Further details regarding the patch can be viewed at the [pull request URL]({{ s
 The patch has been released as versions `3.6.3`, `3.7.4` and `3.8.4`.<br/>
 `v3.7.4` was released a couple of weeks prior and has been bundled with `github-pages-v192`.
 
-We recommend that you upgrade to the patch-release for the series used in your site, at the earliest.
+Please keep in mind that this issue affects _all previously released Jekyll versions_. If you have not had
+a good reason to upgrade to `3.6/7/8` yet, we advise that you do so at your earliest.
```

Comment:
3.6, 3.7 or 3.8

Reply:
@DirtyF Can you please push a correction here..

```diff
- we advise that you do so at your earliest.
+ we advise that you do so at the earliest.
```